rainforest/rainforest.go

```diff
@@ -113,7 +123,30 @@ func (c *Client) NewRequest(method, urlStr string, body interface{}) (*http.Requ
 
 	// Set UserAgent header with appended library version, will look like:
 	// "rainforest-cli/2.1.0 [rainforest golang lib/2.0.0]"
-	composedUserAgent := c.UserAgent + " [rainforest golang lib/" + libVersion + "]"
+	userAgent := []string{"rainforest", "golang", "lib/" + libVersion}
+
+	if c.SendTelemetry {
```

Comment:
Sending this info in the user agent is going to make parsing it harder over time, is there a reason to not send it via `X-` headers? 

This would also make this code simpler as we could just do:
```
if c.SendTelemetry {
  found, ci_name := detectci.WhichCI()
  if found {
    req.Header.Set('X-RF-CI-ENV', ci_name)
  }
  ...
}
```

Reply:
@jbarber as I want to get it done, and see if it works; if parsing it out via stack driver is too bad, I'll end up changing this to a header, or some other-way and storing it more explicitly - not just invthe log.

Reply:
Ah, I thought you were going to do the parsing in the main app. If you've only got access to the User-Agent, then sticking it there makes sense.

Reply:
@jbarber yah - I figured getting this out, then using that first would be easiest. If it's crap, I'll move it.